content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -252,6 +252,13 @@ kubectl get csr/john -o yaml
 
 The certificate value is in Base64-encoded format under `status.certificate`.
 
+Export the issued certificate from the CertificateSigningRequest.
+This example assumes the key and certificate files are located at "/home/vagrant/work/".
```

Comment:
suggest dropping the `/home/vagrant` paths from the example. The creation of the `john.key` and `john.csr` files above just uses the current working directory.

Reply:
Sounds good.  Done.